File: src/core/udp/constants/udp-host-identity-constants.ts
// Identity the dedicated host presents on the p2p channel.
//
// peer_id on the wire is the SENDER's own id: the reply must carry the HOST's
// character id (the seeded NPC, 1) — the joiner's drain gate compares it
// against the peer descriptor stored in its dial session, NOT an echo of its
// own id (§4 gate 1).
export const UDP_HOST_PEER_ID = 1;
export const UDP_HOST_COUNTER_BASE = 0x12345678;

File: src/Shared/Constants/UdpHostIdentityConstants.cs
namespace Mgo2Server.Shared.Constants;

/// <summary>
/// Identity the dedicated gameplay host presents on the peer-to-peer channel.
/// </summary>
public static class UdpHostIdentityConstants
{
    /// <summary>
    /// The peer identifier on the wire is the sender's own identifier: the
    /// handshake reply must carry the host's character identifier, because the
    /// joining client's drain gate compares it against the peer descriptor
    /// stored in its dial session rather than echoing its own identifier.
    /// </summary>
    public const uint HostPeerIdentifier = 1;

    /// <summary>Counter base the host announces in its handshake.</summary>
    public const uint HostCounterBase = 0x12345678;
}
